joomla/joomla-cms
# Add alternative layout parameter to com_content menutypes

### Summary of Changes
This PR adds the possibility to define a specific Alternative Layout for the Article which is displayed in a:
1) Category List Menutype
2) Blog View Menutype
3) List all Categories Menutype

### Testing Instructions
Create an Alternate Layout for your article and create these three Menütypes
1) Category List Menutype
2) Blog View Menutype
3) List all Categories Menutype

Select in each Menutype your alternate layout and see that the layout settings on the Articles that are called FROM this views are applied.
